ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/title/clientlibs/css.txt

```diff
@@ -0,0 +1 @@
+less/title.less
```

Comment:
@rakeshkumar-verma @ashishgoyalweb : what are the final variations for title ?

Reply:
As per Acceptance criteria as of now will have H1 to H6 styles.

Reply:
@rakeshkumar-verma : will need the p tag too for this.

Reply:
default style added to p tag

Reply:
added p Tag